packages/dev-approuter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dev-approuter",
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
@nicoschoenteich : you do not need to increase the version manually, this will happen with the release build!

Reply:
Ok